base/v0_5_exp/translate.go

```diff
@@ -217,6 +218,85 @@ func translateLink(from Link, options common.TranslateOptions) (to types.Link, t
 	return
 }
 
+func translatePasswdUser(from PasswdUser, options common.TranslateOptions) (to types.PasswdUser, tm translate.TranslationSet, r report.Report) {
+	tr := translate.NewTranslator("yaml", "json", options)
+	tm, r = translate.Prefixed(tr, "gecos", &from.Gecos, &to.Gecos)
+	translate.MergeP(tr, tm, &r, "groups", &from.Groups, &to.Groups)
+	translate.MergeP(tr, tm, &r, "home_dir", &from.HomeDir, &to.HomeDir)
```

Comment:
`MergeP` doesn't automatically perform the necessary `camelCase` conversion, so you need `MergeP2` for fields with underscores.